### ISSUE TITLE
Sequence of quad points incorrect after extraction via `Page.get_drawings()`

### ISSUE THREAD

**Issue by jn-chrn** [pymupdf/PyMuPDF#1583]
## Describe the bug
Some drawings are considered not rectangular when extracted using `Page.get_drawings()` even though they are. Indeed, the property `is_rectangular` of the obtained `Quad` returns `False` instead of `True`.

By taking a look into `is_rectangular`, I could see that the sine computation returned a value of `-1.0`, which should be considered rectangular. However the property returns then `False` because of this value.

## To Reproduce
To reproduce it, you can use the following snippet which:
 - creates a PDF with a single page and a square quad as single drawing;
 - extracts the drawings from the created PDF document;
 - gets the value of the `is_rectangular` property of the single `Quad` found.

```python
import fitz

# A square of width/height of sqrt(2) * 10
quad = fitz.Quad(
    fitz.Point(20.0, 10.0),
    fitz.Point(30.0, 20.0),
    fitz.Point(10.0, 20.0),
    fitz.Point(20.0, 30.0),
)

# Write the PDF
out_pdf = fitz.open()
out_page = out_pdf.new_page(width=50, height=50)
shape = out_page.new_shape()

shape.draw_quad(quad)
shape.finish(color=(0, 0, 0))
shape.commit()

# Read the square from the PDF
drawings = out_page.get_drawings()
items = drawings[0]["items"]
_, read_quad = items[0]
assert isinstance(read_quad, fitz.Quad)

# The Quad should be rectangular, but is not
print(read_quad.is_rectangular)
```

## Expected behavior
The `Quad` object created in the snippet is a square rotated with angle 90°. It should thus be computed as being rectangular.

## Screenshots
A screenshot of the PDF created above in the snippet, with a clearly rectangular shape.
![Screenshot from 2022-02-03 16-34-07](https://user-images.githubusercontent.com/98965709/152374745-eed0dc75-c18f-47eb-baf9-833b6512d021.png)

## Your configuration 

Result of the command:

```
3.8.12 (default, Oct 12 2021, 13:49:34) 
[GCC 7.5.0] 
 linux 
 
PyMuPDF 1.19.4: Python bindings for the MuPDF 1.19.0 library.
Version date: 2022-01-01 00:00:01.
Built for Python 3.8 on linux (64-bit).
```
PyMuPDF is installed inside a conda environment with pip.

**Comment by JorjMcKie** [comment]
Cause of error found and corrected. Will be fixed in coming version.
Thanks again for reporting this.

**Comment by JorjMcKie** [comment]
Fixed in v1.19.5 currently being uploaded.

**Comment by jn-chrn** [comment]
Thank you for handling it so fast!
